README.md

```diff
@@ -16,7 +16,9 @@ cargo install --version 0.1.1 --git https://github.com/ElektraInitiative/mdbook-
 mdbook serve --open
 ```
 
-Note that Rust needs to be installed for it to work.
+Important links:
```

Comment:
Maybe write a section documentation like in [Actix](https://github.com/actix/actix-web#documentation)?
Then it seems clearer what additional information there is (and we can add the code doc and mdbook websites as well once they are deployed)

Reply:
I don't understand, it already is in a section Documentation? The self-building of mdbook is only as long as the CI does not do it, eventually in the section should be only links, like in Actix.

Reply:
Oh sorry. That's my bad. Github just cut out the Documentations section and I didn't realize. So all is good here.